components/HackathonTimeline.jsx

```diff
@@ -0,0 +1,63 @@
+import Heading from './Heading';
+import Image from 'next/image';
+import { UnderlineTypes } from '../utils/underlineType';
+import { HACKATHONS } from '../data/hackathons';
+
+const HACKATHON_TIMELINE_CONTAINER = 'flex justify-center bg-white';
+const HACKATHON_TIMELINE_CONTENT = 'max-w-[500px] w-[80vw] md:max-w-[1100px]';
+const HACKATHON_TIMELINE_HEADER = 'md:justify-start md:my-[60px] my-[30px] flex justify-center';
+const HACKATHON_EVENT_INFO = 'flex text-center text-xl font-bold mb-[70px] md:items-center flex-col items-center';
+const HACKATHON_TITLE_YEAR = 'text-5xl pb-[15px] italic md:text-4xl';
+const HACKATHON_IMAGE = 'object-cover w-[80vw] md:rounded-xl md:w-[450px] md:h-[250px] md:shadow-[15px_15px_0px_0px_#00D3A9]';
+const HACKATHON_TEXT = 'flex-col md:w-[400px] md:mx-[40px] leading-[1] text-[20px]';
+const HACKATHON_TOPIC_TEXT = 'flex text-left my-[15px] text-[#FF4D6F] md:text-[#FF6B54] md:my-1 md:font-extrabold md:text-[25px]';
+
+const HackathonSection = ({ hackathon, index }) => {
+  let sectionInfoStyling = '';
```

Comment:
![image](https://github.com/Code-the-Change-YYC/code-the-change-yyc-site/assets/113923596/64eaa898-d6d3-4148-9752-65f1d91e44f2)
Consider this coding style.

Reply:
Oh yea, that's much more concise, thank you!